PROVISIONING-METAL.md

```diff
@@ -34,11 +34,12 @@ sha512sum -c <<<"a3c58bc73999264f6f28f3ed9bfcb325a5be943a782852c7d53e803881968e0
 
 Next, set your desired version and variant, and use `tuftool` to download the image:
 To install `tuftool` you'll need to install Rust (via [rustup](https://rustup.rs/) or the official site), and then you can run `cargo install tuftool`.
+The VERSION corresponds to the [Bottlerocket version](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.16.1), the latest release is almost always what you want.
 
 ```shell
 ARCH="x86_64"
-VERSION="v1.12.0"
-VARIANT="metal-k8s-1.23"
+VERSION=$(curl -s https://api.github.com/repos/bottlerocket-os/bottlerocket/releases/latest | jq -r '.name')
+VARIANT="metal-k8s-1.28"
```

Comment:
Use of `jq` implies a non-standard prereq, but I think for this audience that would be OK.

Reply:
This is a good callout. We mention installing `jq` for the quickstarts and I assumed we had called that out here too. I can add a line saying you might need to install it to use that one-liner.